spec/services/container_topology_service_spec.rb

```diff
@@ -1,11 +1,14 @@
 require "spec_helper"
 
 describe ContainerTopologyService do
+
   let(:container_topology_service) { described_class.new(nil) }
 
   describe "#build_kinds" do
     it "creates the expected number of entity types" do
-      container_topology_service.build_kinds.size.should eql 8
+      ems_kube = FactoryGirl.create(:ems_kubernetes, :name => "ems_kube")
```

Comment:
> Also, avoid setting name if you can. The factories will create unique names for avoid name collisions, and you can always pull the name back out if you need them for comparison later.

why does it matter if I set a name? (I definitely saw it in several other places in miq)